### PR TITLE
Rename everything

### PR DESCRIPTION
**Pull request by pstalcup**

### CODE REVIEW

File: BUGS.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ Try to make sure you have a stable internet connection, and that your device won
...
+Garbo is a big, complicated script that often finds itself doing big, complicated things. While its express purpose is to farm meat at Barf Mountain, it can do a lot of unconventional-seeming things along the way. It will likely try to fight a large number of target monsters (usually Cheerless Mime Executive or Knob Gobline Elite Captain) at the start of the day. When it fights "wandering" monsters, like those found using Digitize, it will try to place those monsters into special zones from which it can eke additional value. It will pause throughout the day to Spit Jurassic Acid in a variety of zones, assuming you have a Jurassic Parka. This list is extremely, extremely non-exhaustive--KoL is a 20 year old game with profoundly complex interacting mechanics, and Garbo exploits a hell of a lot of them.
```
Comment (Tokoeka) [inline BUGS.md:22]: /Goblin/ not /Gobline/

File: BUGS.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ Try to make sure you have a stable internet connection, and that your device won
...
 
 Given all this, how can we know that something is a bug? We have a few rules of thumb:
 
-- In general, if Garbo is outside barf mountain and is not fighting a Knob Goblin Embezzler, it should probably not be spending turns.
+- In general, if Garbo is outside barf mountain and is not fighting a target monster (usually Cheerless Mime Executive or Knob Gobline Elite Captain), it should probably not be spending turns.
```
Comment (Tokoeka) [inline BUGS.md:26]: /Goblin/ not /Gobline/

File: README.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ To invoke Garbage Collector, type `garbo` in the Mafia GCLI. In addition, you ca
 
 ### `nobarf` flag
 
-Running Garbo in `nobarf` mode will not target spending the bulk of your turns at Barf Mountain. Instead, it will just do your daily flag turns, as well as attempt to fight as many Knob Goblin Embezzlers as possible.
+Running Garbo in `nobarf` mode will not target spending the bulk of your turns at Barf Mountain. Instead, it will just do your daily flag turns, as well as attempt to fight as many target monsters (usually Cheerless Mime Executive or Knob Gobline Elite Captain) as possible.
```
Comment (Tokoeka) [inline README.md:101]: /Goblin/ not /Gobline/

File: packages/garbo/src/lib.ts [hunk 4]
```diff
@@ -383,7 +382,7 @@ export function pillkeeperOpportunityCost(): number {
   const alternateUses = [
     {
       can: canTreasury,
-      value: EMBEZZLER_MULTIPLIER() * get("valueOfAdventure"),
+      value: TARGET_MULTIPLIER() * get("valueOfAdventure"),
```
Comment (Tokoeka) [inline packages/garbo/src/lib.ts:385]: This replace of Embezzler feels wrong under the current paradigm given this is specifically accounting for targetting embezzlers via lucky?

Or is the (future) intent to include embezzler calcs within target_multiplier()?

**Review by Tokoeka — COMMENTED** [review]
Some places have replaced "embezzler" with "target", others with "meatTarget". I assume this was split on purpose, but wanted to check

**Comment by horrible-little-slime** [comment]
I'm not convinced that calculating the former-embezzlermultiplier is a good idea. There's a reason we split it off into a pref! A lot of our value doesn't come from meat, it comes from familiar drops and weird shit like the LGR and from pointer finger crits

File: packages/garbo/src/familiar/barfFamiliar.ts [hunk 5]
```diff
@@ -63,7 +63,7 @@ const SPECIAL_FAMILIARS_FOR_CACHING = new Map<
     $familiar`Chest Mimic`,
     {
       extraValue: ({ famexp }) =>
-        (famexp * EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")) / 50,
+        (famexp * MEAT_TARGET_MULTIPLIER() * get("valueOfAdventure")) / 50,
```
Comment (Tokoeka) [inline packages/garbo/src/familiar/barfFamiliar.ts:66]: Possibly not something for this PR, but do we need to make some adjustment here for if the copy target is not a meat_target

File: packages/garbo/src/outfit/barf.ts [hunk 6]
```diff
@@ -124,7 +124,8 @@ export function computeBarfOutfit(
     outfit.familiar === $familiar`Chest Mimic` &&
     $familiar`Chest Mimic`.experience < 550
   ) {
-    const famExpValue = (EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")) / 50;
+    const famExpValue =
+      (MEAT_TARGET_MULTIPLIER() * get("valueOfAdventure")) / 50;
```
Comment (Tokoeka) [inline packages/garbo/src/outfit/barf.ts:128]: Similarily, do we need something here in the case where the copy target isnt a meat target?

File: packages/garbo/src/outfit/free.ts [hunk 7]
```diff
@@ -47,7 +47,7 @@ export function freeFightOutfit(
       [
         [
           $familiar`Chest Mimic`,
-          (EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")) / 50,
+          (MEAT_TARGET_MULTIPLIER() * get("valueOfAdventure")) / 50,
```
Comment (Tokoeka) [inline packages/garbo/src/outfit/free.ts:50]: And again here?

File: packages/garbo/src/tasks/barfTurn.ts [hunk 8]
```diff
@@ -355,7 +355,7 @@ const NonBarfTurnTasks: AlternateTask[] = [
       }
       ChestMimic.differentiate(globalOptions.target);
     },
-    outfit: () => embezzlerOutfit({ familiar: $familiar`Chest Mimic` }),
+    outfit: () => meatTargetOutfit({ familiar: $familiar`Chest Mimic` }),
```
Comment (Tokoeka) [inline packages/garbo/src/tasks/barfTurn.ts:358]: Again, is it worth adding in something in the case of freefight target vs meat target vs [other] target?

File: packages/garbo/src/tasks/barfTurn.ts [hunk 9]
```diff
@@ -833,7 +833,7 @@ const BarfTurnTasks: GarboTask[] = [
     },
     combat: new GarboStrategy(() => Macro.meatKill()),
     spendsTurn: () => globalOptions.target.attributes.includes("FREE"),
-    outfit: () => embezzlerOutfit(),
+    outfit: () => meatTargetOutfit(),
```
Comment (Tokoeka) [inline packages/garbo/src/tasks/barfTurn.ts:836]: do we need an alternate outfit option for non-meat targets?

File: packages/garbo/src/tasks/barfTurn.ts [hunk 10]
```diff
@@ -847,15 +847,15 @@ const BarfTurnTasks: GarboTask[] = [
     },
     combat: new GarboStrategy(() => Macro.meatKill()),
     spendsTurn: () => globalOptions.target.attributes.includes("FREE"),
-    outfit: () => embezzlerOutfit({ familiar: $familiar`Chest Mimic` }),
+    outfit: () => meatTargetOutfit({ familiar: $familiar`Chest Mimic` }),
```
Comment (Tokoeka) [inline packages/garbo/src/tasks/barfTurn.ts:850]: same query here?

File: packages/garbo/src/tasks/barfTurn.ts [hunk 11]
```diff
@@ -847,15 +847,15 @@ const BarfTurnTasks: GarboTask[] = [
...
   },
   {
     name: "Fight Mimic Eggs",
     ready: () => globalOptions.ascend,
     completed: () =>
       ChestMimic.differentiableQuantity(globalOptions.target) === 0,
     do: () => ChestMimic.differentiate(globalOptions.target),
-    outfit: () => embezzlerOutfit(),
+    outfit: () => meatTargetOutfit(),
```
Comment (Tokoeka) [inline packages/garbo/src/tasks/barfTurn.ts:858]: and here?

File: packages/garbo/src/outfit/barf.ts (hunk 6 above)
Comment (horrible-little-slime) [reply packages/garbo/src/outfit/barf.ts:128]: I think for now this obeys the Hippocratic oath

**Comment by horrible-little-slime** [comment]
LGTM but I need to resolve merge conflicts and it's not on loathers

**Pull request closed without merging**